bots/hermes/cron_script.py

```diff
@@ -0,0 +1,25 @@
+import dbstore
+from telegram import Bot
+
+''' Script to be run from a cron job, to fetch recent projects and send to registed 'chats' '''
+
+bot = Bot(token='TOKEN')
```

Comment:
Can you use config parser to get this token ?

Reply:
Maybe take some inspiration from that 
https://github.com/osscameroon/osscameroon-website/blob/main/api/app/settings.py